### PR TITLE
New topic on hot springs

### PR DESCRIPTION
**Pull request by Daiozen**

### CODE REVIEW

File: Monika After Story/game/script-topics.rpy [hunk 1]
```diff
@@ -14195,3 +14195,46 @@ label monika_translating_poetry:
...
+label monika_hot_spring:    
+    m 1eua "Have you ever been to a hot spring, [player]?"
+    m 3eub "I've never been to one myself, but it's something I'd like to try when I get here."
+    m 3dka "They're supposed to be a great way to relieve stress and rest a little."
+    m 3hub "Not to mention the many health benefits!"
+    m 4eub "They help you get good blood circulation for one. There's also some minerals in the water that can help boost your immune system, too!"
+    m 7eua "I'd like to try an open-air bath in particular. I hear it really gives a different experience."
+    m 7rubssdla "Though it might feel a little weird chilling in a bath with that many people all around you..."
+    m 1hkblsdlb "It sounds a little embarassing~" 
+    if (mas_isMoniEnamored(higher=True) and (renpy.random.randint(1,4) == 1)):
```
Comment (multimokia) [inline Monika After Story/game/script-topics.rpy:14219]: Don't need the extra brackets here.
```suggestion
    if mas_isMoniEnamored(higher=True) and renpy.random.randint(1,4) == 1:
```

File: Monika After Story/game/script-topics.rpy [hunk 2]
```diff
@@ -14195,3 +14195,46 @@ label monika_translating_poetry:
...
+        m 1rkbssdlu "..."
+        m 3rkbfsdlb "Especially since some places don't allow you to wear any sort of cover either!"
+        m 3rubfb "Fortunately, most of these places don't allow for mixed bathing anymore..."
+        m 1tubfu "Well, maybe I wouldn't mind it so much if it was with you..."
+    m 1tubsb "Maybe we could find a place where we could be just the two of us?" 
+    show monika 5ekbfa at t11 zorder MAS_MONIKA_Z with dissolve
+    m 5ekbfa "Can you imagine how it'd be? You and me relaxing in a nice, soothing hot pool..."
```
Comment (multimokia) [inline Monika After Story/game/script-topics.rpy:14226]:
```suggestion
    m 5ekbfa "Can you imagine how it'd be? You and me relaxing in a nice, soothing hot pool..."

```

File: Monika After Story/game/script-topics.rpy [hunk 3]
```diff
@@ -14195,3 +14195,46 @@ label monika_translating_poetry:
...
+    m 3hub "Not to mention the many health benefits!"
+    m 4eub "They help you get good blood circulation for one. There's also some minerals in the water that can help boost your immune system, too!"
+    m 7eua "I'd like to try an open-air bath in particular. I hear it really gives a different experience."
+    m 7rubssdla "Though it might feel a little weird chilling in a bath with that many people all around you..."
+    m 1hkblsdlb "It sounds a little embarassing~" 
+    if (mas_isMoniEnamored(higher=True) and (renpy.random.randint(1,4) == 1)):
+        m 1rkbssdlu "..."
+        m 3rkbfsdlb "Especially since some places don't allow you to wear any sort of cover either!"
+        m 3rubfb "Fortunately, most of these places don't allow for mixed bathing anymore..."
+        m 1tubfu "Well, maybe I wouldn't mind it so much if it was with you..."
```
Comment (multimokia) [inline Monika After Story/game/script-topics.rpy:14223]:
```suggestion
        m 1tubfu "Well, maybe I wouldn't mind it so much if it was with you..."

```

File: Monika After Story/game/script-topics.rpy [hunk 4]
```diff
@@ -14195,3 +14195,46 @@ label monika_translating_poetry:
...
+    m 1tubsb "Maybe we could find a place where we could be just the two of us?" 
+    show monika 5ekbfa at t11 zorder MAS_MONIKA_Z with dissolve
+    m 5ekbfa "Can you imagine how it'd be? You and me relaxing in a nice, soothing hot pool..."
+    if (mas_isWinter()):
```
Comment (multimokia) [inline Monika After Story/game/script-topics.rpy:14227]:
```suggestion
    if mas_isWinter():
```

File: Monika After Story/game/script-topics.rpy [hunk 5]
```diff
@@ -14195,3 +14195,46 @@ label monika_translating_poetry:
...
+       m 5dubfu "Warming our chilled bodies after a harsh day out in the cold..."
+    elif (mas_isSummer()):
```
Comment (multimokia) [inline Monika After Story/game/script-topics.rpy:14229]:
```suggestion
    elif mas_isSummer():
```

File: Monika After Story/game/script-topics.rpy [hunk 6]
```diff
@@ -14195,3 +14195,46 @@ label monika_translating_poetry:
...
+       m 5dubfu "Wiping the sweat off after a long day out in the sun..."
+    elif (mas_isFall()):
```
Comment (multimokia) [inline Monika After Story/game/script-topics.rpy:14231]:
```suggestion
    elif mas_isFall():
```

File: Monika After Story/game/script-topics.rpy [hunk 7]
```diff
@@ -14195,3 +14195,46 @@ label monika_translating_poetry:
...
+       m 5dubfu "Watching the leaves gently fall around us in the last lights of the afternoon..."
+    elif (mas_isSpring()):
```
Comment (multimokia) [inline Monika After Story/game/script-topics.rpy:14233]: Don't need to explicitly check for spring here
```suggestion
    else:
```

File: Monika After Story/game/script-topics.rpy [hunk 8]
```diff
@@ -14195,3 +14195,46 @@ label monika_translating_poetry:
...
+       m 5dubfu "Contemplating the beauty of nature all around us..."
```
Comment (multimokia) [inline Monika After Story/game/script-topics.rpy:14234]:
```suggestion
       m 5dubfu "Contemplating the beauty of nature all around us..."

```

File: Monika After Story/game/script-topics.rpy [hunk 9]
```diff
@@ -14195,3 +14195,46 @@ label monika_translating_poetry:
...
+    if (mas_isMoniEnamored(higher=True) and (renpy.random.randint(1,4) == 1)):
+        m 1rkbssdlu "..."
+        m 3rkbfsdlb "Especially since some places don't allow you to wear any sort of cover either!"
+        m 3rubfb "Fortunately, most of these places don't allow for mixed bathing anymore..."
+        m 1tubfu "Well, maybe I wouldn't mind it so much if it was with you..."
+    m 1tubsb "Maybe we could find a place where we could be just the two of us?" 
+    show monika 5ekbfa at t11 zorder MAS_MONIKA_Z with dissolve
+    m 5ekbfa "Can you imagine how it'd be? You and me relaxing in a nice, soothing hot pool..."
+    if (mas_isWinter()):
+       m 5dubfu "Warming our chilled bodies after a harsh day out in the cold..."
```
Comment (multimokia) [inline Monika After Story/game/script-topics.rpy:14228]: Inconsistent whitespace 
```suggestion
        m 5dubfu "Warming our chilled bodies after a harsh day out in the cold..."
```

File: Monika After Story/game/script-topics.rpy [hunk 10]
```diff
@@ -14195,3 +14195,46 @@ label monika_translating_poetry:
...
+    elif (mas_isSummer()):
+       m 5dubfu "Wiping the sweat off after a long day out in the sun..."
```
Comment (multimokia) [inline Monika After Story/game/script-topics.rpy:14230]:
```suggestion
        m 5dubfu "Wiping the sweat off after a long day out in the sun..."
```

File: Monika After Story/game/script-topics.rpy [hunk 11]
```diff
@@ -14195,3 +14195,46 @@ label monika_translating_poetry:
...
+    elif (mas_isFall()):
+       m 5dubfu "Watching the leaves gently fall around us in the last lights of the afternoon..."
```
Comment (multimokia) [inline Monika After Story/game/script-topics.rpy:14232]:
```suggestion
        m 5dubfu "Watching the leaves gently fall around us in the last lights of the afternoon..."
```

File: Monika After Story/game/script-topics.rpy (hunk 8 above)
Comment (multimokia) [inline Monika After Story/game/script-topics.rpy:14234]:
```suggestion
        m 5dubfu "Contemplating the beauty of nature all around us..."
```

File: Monika After Story/game/script-topics.rpy [hunk 12]
```diff
@@ -14195,3 +14195,48 @@ label monika_translating_poetry:
...
+            eventlabel="monika_hot_spring",
+            category=['nature','misc'],
+            prompt="Hot springs",
+            random=True
+        )
+    )
+
+label monika_hot_spring:    
+    m 1eua "Have you ever been to a hot spring, [player]?"
+    m 3eub "I've never been to one myself, but it's something I'd like to try when I get here."
```
Comment (jmwall24) [inline Monika After Story/game/script-topics.rpy:14212]:
```suggestion
    m 3eub "I've never been to one myself, but it's something I'd like to try when I get to your world."
```

File: Monika After Story/game/script-topics.rpy [hunk 13]
```diff
@@ -14195,3 +14195,48 @@ label monika_translating_poetry:
...
+    m 3dka "They're supposed to be a great way to relieve stress and rest a little."
+    m 3hub "Not to mention the many health benefits!"
```
Comment (jmwall24) [inline Monika After Story/game/script-topics.rpy:14214]:
```suggestion
    m 3hub "...Not to mention the many health benefits!"
```

**Comment by jmwall24** [comment]
Can you add a line or 2 just that serves as a disclaimer, or perhaps at least word it a little differently? This topic is implying that all hot springs are places where you can go and bathe in, when in fact, many of them are either too hot to even touch and/or are extremely acidic and would dissolve you. A line about only going in the ones that are actually designated for public swimming/bathing and to always follow all posted rules would likely be enough. I just don't want someone to see this and go running to their nearest hot spring and dying.

**Comment by Daiozen** [comment]
Thanks for the input, it hadn't even occured to me it could be interpreted this way.
This last commit should fix this.

**Pull request merged**